<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="TXT">
  <testsuite name="Cloning TXT file and verifying" tests="6" failures="1" errors="0" skipped="0" time="6.1">
    <testcase name="test 1" time="0.1" classname="test.java.PlayingWithExcelFiles" />
    <testcase name="test 2" time="0.7" classname="test.java.PlayingWithExcelFiles" />
    <testcase name="test 3" time="0.9" classname="test.java.PlayingWithExcelFiles" />
    <testcase name="test 4" time="1" classname="test.java.PlayingWithExcelFiles" />
    <testcase name="test 5" time="3" classname="test.java.PlayingWithExcelFiles" >
      <failure type="java.lang.AssertionError" message="Test failed as 'pass' was not found expected [true] but found [false]"><![CDATA[java.lang.AssertionError: Test failed as 'pass' was not found expected [true] but found [false]
    org.testng.Assert.fail(Assert.java:94)
    org.testng.Assert.failNotEquals(Assert.java:496)
    org.testng.Assert.assertTrue(Assert.java:42)
    test.java.PlayingWithExcelFiles.verifyingTheClonedFile(PlayingWithExcelFiles.java:35)
    sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
    org.testng.TestNG.run(TestNG.java:1032)
    org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
    org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
    org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
    org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
    org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
    org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)]]></failure>
  </testcase>
    <testcase name="test 6" time="0.4" classname="test.java.PlayingWithExcelFiles" />
  </testsuite>
  <testsuite name="Alternate test suite">
    <testcase name="alternate test 1" time="0.1" classname="test.java.AlternateTestSuite" />
    <testcase name="alternate test 2" time="1.900" classname="test.java.AlternateTestSuite" >
      <failure type="java.lang.AssertionError" message="Test failed as 'pass' was not found expected [true] but found [false]"><![CDATA[java.lang.AssertionError: Test failed as 'pass' was not found expected [true] but found [false]
    org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)]]></failure>
    </testcase>
  </testsuite>
</testsuites>

